packages/nextjs/components/scaffold-stark/Balance.tsx

```diff
@@ -52,33 +62,49 @@ export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
     );
   }
 
-  //const formattedBalance = balance ? Number(balance.formatted) : 0;
+  // Calculate the total balance in USD
+  const ethBalanceInUsd = parseFloat(formatted) * price;
+  const strkBalanceInUsd = parseFloat(strkBalance.formatted) * price;
```

Comment:
why are we using the same price for Eth and Strk ?

Reply:
> why are we using the same price for Eth and Strk ?

@0xquantum3labs  From here you can see that the comment on the review asked for the  sum of the value of both tokens

https://github.com/Quantum3-Labs/scaffold-stark-2/pull/226#issuecomment-2264788700

![image](https://github.com/user-attachments/assets/b95cc2cc-c8e2-4ab0-a902-9635beaa53a9)

Reply:
the value of ETH and STRK are not the same @raizo07

Reply:
the price of STRK and the price of ETH are not the same
@raizo07

Reply:
@raizo07 gm any updates ?

Reply:
Hello @0xquantum3labs  the targetNetwork here always returns Eth.

How do I get it to return Strk

![image](https://github.com/user-attachments/assets/b8d21161-8240-41a8-9056-8d155439e865)

Reply:
you can create another hook `useStrkCurrencyPrice()` and adapt the  `fetchPriceFromCoinGecko()` function